src/gt4py/gtc/gtir.py

```diff
@@ -206,10 +320,10 @@ def build(self) -> FieldsMetadata:
         return FieldsMetadata(metas={k: v.build() for k, v in self.metas.items()})
 
 
-class Computation(LocNode):
-    name: Str
-    params: List[FieldDecl]
-    stencils: List[Stencil]
+class Stencil(LocNode):
+    name: SymbolName.constrained(r"[a-zA-Z_][\w\.]*")
```

Comment:
TODO remove `__main__.` from name
```suggestion
    name: SymbolName
```

Reply:
I added a TODO for later